tests/_support/Page/Framework.php

```diff
@@ -1129,4 +1129,18 @@ public function iUploadTheItemTypeCASEFile()
         return $this;
     }
 
+    /**
+     * @When /^I download the framework PDF file$/
+     */
+    public function iDownloadTheFrameworkPdfFile(): Framework
+    {
+        $I = $this->I;
+
+        $I->click('Export', '#itemInfo');
+        $I->waitForElementVisible('#exportModal a.btn-export-pdf');
+        $I->click('//*[@id="exportModal"]/div/div/div[2]/div/div[1]/div/div[3]/a');
```

Comment:
This makes sure that the button is there to click, but there should be a check that a file is downloaded (and that it is a PDF file, possibly also that the file contains what is expected).

Also, to make it easier to read, you might want to use the same CSS selector that is used in the `waitForElementVisible` call instead of using the XPath selector.